src/uct/api/v2/uct_v2.h

```diff
@@ -554,6 +554,13 @@ typedef struct uct_iface_is_reachable_params {
      * This value must be specified in conjunction with @a info_string.
      */
     size_t                        info_string_length;
+
+    /**
+     * Output parameter; Indicates whether both addresses describe the same
+     * device.
+     */
+    int                           is_self;
```

Comment:
do we need to update `uct_iface_is_reachable_field_mask_t`?

Reply:
is it relevant for output params?

Reply:
i'd expect so

Reply:
IMO using the word `self` is problematic, as it causes confusion with the `self` transport.
Maybe `is_same_device` instead?